xml/ay_bigfile.xml

```diff
@@ -2473,6 +2473,10 @@ openssl x509 -noout -fingerprint -sha256
           <filename>/dev/disk/by-path/pci-0001:00:03.0-scsi-0:0:0:0</filename>.
          </para>
 <screen>&lt;device&gt;/dev/sda&lt;/device&gt;</screen>
+        <para>
+          In case of volume groups, software RAID or bcache devices, the name in the installed
```

Comment:
The documentation defines the `&bcache;` entity, use it instead "bcache" literal. (See https://github.com/SUSE/doc-sle/blob/develop/xml/entity-decl.ent#L218)